modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CppPistacheServerCodegen.java

```diff
@@ -43,7 +43,9 @@
 
 public class CppPistacheServerCodegen extends AbstractCppCodegen {
     protected String implFolder = "impl";
-
+    protected boolean isAddExternalLibs = true;
```

Comment:
@stkrwork 
Should we make this default false.
Normally when it is true we can build it out of the box.

Reply:
@etherealjoy I suggest either merging it to 3.2.x (breaking changes without fallback) if defaults to `true` or change the default to false if we want to merge this PR into current master.